### ISSUE TITLE
Is PPA defunct for 24.10? If so, modify web page?

### ISSUE THREAD

**Issue by chconnor** [keepassxreboot/keepassxc#11416]
Hi!

It would seem that keepassxc is now a package in the main *ubuntu repositories, so the PPA is no longer needed, is that correct?

If so, perhaps the [download page](https://keepassxc.org/download/#linux) could be modified to make that clear?

It offers the AppImage, and then the next thing is "other official linux packages" which provides the PPA, but it's useless in 24.10. Only under "See more options" do we see the official repo install instructions, which should be at or near the top, IMO.

Just a thought -- would have saved me some time. Thanks!

**Comment by droidmonkey** [comment]
Always check your native repo option first. That is the "download" page after all.

**Comment by chconnor** [comment]
Sure, it's just that it's been installed by repo previously, so I went to do that first... and as you know very many projects provide less updated versions via official repo and latest versions via PPA, and I assumed the same was happening here, especially because the PPA is in the "Other Official Linux Packages" section and the repo mention is in the "See more options" which makes it sound like the less desirable option, especially for something security-related.

TLDR: it might save some people some grief if instead of "Ubuntu PPA" it said "Ubuntu PPA (24.04 and earlier)".

**Comment by droidmonkey** [comment]
> very many projects provide less updated versions via official repo

Projects don't control what is in the official repo. That is up to the repo managers. It just so happens Ubuntu 24.10 included our latest version, but that won't likely be updated anytime soon after our next release. We (nor most projects) can control the update pace of Ubuntu.

That is why PPA's exist, among other reasons.

**Comment by chconnor** [comment]
Thanks -- do you have plans to update the PPA when that happens, or would it be best to move to the AppImage at that point?

**Comment by droidmonkey** [comment]
The ppa is updated on every release. We ultimately recommend flatpak as it offers a sandbox environment and auto updates

**Comment by phoerious** [comment]
I've uploaded a version for Oracular. Should be available in half an hour or so.
